docs/howtos/runtime_sessions.rst

```diff
@@ -0,0 +1,70 @@
+Use Experiments with Runtime sessions
+=====================================
+
+Problem
+-------
+
+You want to run experiments in a `Runtime session
+<https://qiskit.org/ecosystem/ibm-runtime/sessions.html>`_ so that jobs can run in close temporal proximity.
+
+Solution
+--------
+
+There are two pathways currently supported:
+
+1. Use the :class:`~qiskit_ibm_provider.IBMBackend` object in ``qiskit-ibm-provider``, which supports sessions.
+
+.. jupyter-input::
+
+    from qiskit_ibm_provider import IBMProvider
+    from qiskit_experiments.library.tomography import ProcessTomography
+    from qiskit import QuantumCircuit
+
+    provider = IBMProvider()
+    backend = provider.get_backend("ibm_nairobi")
+    qc = QuantumCircuit(1)
+    qc.x(0)
+
+    with backend.open_session() as session:
+        tomography = ProcessTomography(qc)
+        tomography_result = tomography.run(backend)
+        tomography_result.block_for_results()
+        session.cancel()
+
+2. Use the ``qiskit-ibm-runtime`` provider. This requires extracting the circuits from the
```

Comment:
What is the advantage of this method over the previous one? The previous one seems much cleaner in the current framework.

Reply:
Right, I wanted to show the `circuit-runner` usage pattern but there's no point now that `backend.run()` has been added to the runtime provider. I've removed the second method.